iceoryx_posh/test/moduletests/test_roudi_process.cpp

```diff
@@ -47,7 +47,7 @@ class Process_test : public Test
 {
   public:
     const iox::RuntimeName_t processname = {"TestProcess"};
-    pid_t pid{200U};
+    uint32_t pid{200U};
```

Comment:
Why is this change required? `pid_t` is probably from some of the included headers and signed. (Did not check).

The question is: is it important for subsequent tests that the pid is of type `pid_t`? I do not know. But since the tests do not break for now it is probably ok.